Panic when required configs are missing

Hey there, thank you for submitting an issue!

We are trying to keep issues for feature requests and bug reports. Please
complete the following checklist before creating a new one:

- [x] Is this a **bug report** (if so, is it something you can **debug and fix**?
Send a pull request!)

---

If the `POSTGRES_DB_URL` or corresponding config key is not setup, the program panics. This should exit with error status instead.

```
panic: server configuration error: POSTGRES_DB_URL env variable or config key must be set

goroutine 1 [running]:
main.main()
	/go/src/github.com/datatogether/coverage/server.go:53 +0x36a
```

--- a/server.go
+++ b/server.go
@@ -49,8 +49,8 @@ func main() {
 	var err error
 	cfg, err = initConfig(os.Getenv("GOLANG_ENV"))
 	if err != nil {
-		// panic if the server is missing a vital configuration detail
-		panic(fmt.Errorf("server configuration error: %s", err.Error()))
+		// fatal if the server is missing a vital configuration detail
+		log.Fatal(fmt.Errorf("server configuration error: %s", err.Error()))
 	}
 
 	go func() {

